pkg/connector/connector.go

```diff
@@ -194,7 +254,13 @@ func (c *GoogleWorkspace) ResourceSyncers(ctx context.Context) []connectorbuilde
 	if err == nil {
 		groupMemberService, err := c.getDirectoryService(ctx, directoryAdmin.AdminDirectoryGroupMemberReadonlyScope)
 		if err == nil {
-			rs = append(rs, groupBuilder(groupService, c.customerID, c.domain, groupMemberService, groupProvisioningService))
+			rs = append(rs, groupBuilder(
+				groupService,
+				c.customerID,
+				c.domain,
```

Comment:
<img width="601" alt="image" src="https://github.com/user-attachments/assets/49c34ee7-dfde-4583-a3cf-9da9ce78f8ea">
I am not sure c.domain will have a value. I don't see it being set anymore.